metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/NetworkFunctions.java

```diff
@@ -232,10 +237,15 @@ private static URL toUrl(Object urlObj) {
     if(urlObj == null) {
       return null;
     }
-    try {
-      return new URL(urlObj.toString());
-    } catch (MalformedURLException e) {
-      return null;
+    if(urlObj instanceof String) {
+      try {
+        return new URL(urlObj.toString());
+      } catch (MalformedURLException e) {
```

Comment:
I'm treating a distinction here between a type or syntactic error which is likely a configuration error and should be immediately percolated up versus a semantic error (the data is malformed, but still of the proper type) which should be allowed to pass.  It's not necessarily an error.  You could have a source which contains a `url` field which is not populated or populated with a default value, so you'd expect the processed fields to also be null.